app/models/rent.rb

```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+  verifies :from, :to, presence: true
```

Comment:
I think you ment validates here, don't you?

Reply:
Don't you think we also need to add a validation for the book relation and the user relation too?

Let me recommend a post that explains you how to validate relations. https://railsguides.net/belongs-to-and-presence-validation-rule1/

Tell me if there's something that you don't understand